lib/index.js

```diff
@@ -71,6 +71,9 @@ export default function runContentfulExport (usageParams) {
           }
         )
       }
+      return response
```

Comment:
Yep definitely not for this PR (that's why I mentioned "The Future")

![Giphy](http://media3.giphy.com/media/92zXMobp0mrvy/giphy.gif)